vmad/core/stdlib/operators.py

```diff
@@ -136,12 +136,12 @@ def apl(node, x1, x2):
         return dict(y=x1 ** x2, logx1=logx1)
 
     def vjp(node, _y, x1, x2, logx1):
-        fac = x1 ** (x2 - 1) if x2 != 1 else 1
+        fac = x1 ** (x2 - 1) #if x2 != 1 else 1
```

Comment:
Let's force cast x2 to numpy.array(x2, dtype='f8')?

If x2 is int and x2 - 1 == 0, this raises an error -- I think that's the original reason for the if clause.

Reply:
I was able to work around this, by overriding the error, and then replacing the "inf" which is put in with a zero.